### ISSUE TITLE
Gutenberg 10.9.0 RC1 breaks Site Editor (Beta)

### ISSUE THREAD

**Issue by optimizewp** [WordPress/gutenberg#32789]
## Description

After installing Gutenberg `10.9.0 RC1` on an `FSE` enabled theme, like `TT1 Blocks` or `Armando`, `Site Editor (Beta)` opens with empty content.

Please note that Gutenberg works with no issue to edit post/pages. 

I am aware of this issue https://github.com/WordPress/gutenberg/issues/32785, which is actual on my local testing environment.

## Step-by-step reproduction instructions

1. install Gutenberg 10.9.0 RC1 on a clean install
2. Activate theme `TT1 Blocks` or `Armando`
3. Activate Gutenberg
4. Open site homepage on front-end (or any page)
5. Click on `Edit Site`  and open `Site Editor (Beta)` (`/wp-admin/admin.php?page=gutenberg-edit-site`) 

## Expected behaviour

`Site Editor (Beta)` works as usual
## Actual behaviour

`Site Editor (Beta)` loads no content and there is a console error

## Screenshots or screen recording (optional)

![image](https://user-images.githubusercontent.com/30923700/122446720-463db480-cfa3-11eb-8241-9d134cd89acc.png)

## WordPress information
- WordPress version: tested with `5.7.2`, `5.7.3`, `5.8 beta 1`, `5.8 beta 2`
- Gutenberg version: 10.9.0 RC1
- Are all plugins except Gutenberg deactivated? Yes
- Are you using a default theme (e.g. Twenty Twenty-One)? Yes `TT1 Blocks` and `Armando` 

## Device information
- Device:  Desktop
- Operating system: Windows 10
- Browser: Edge Version 91.0.864.48 (Official build) (64-bit)

**Comment by Mamaduka** [comment]
Thanks for reporting the issue, @optimizewp.

I see an error in the console on the Site Editor page, but it's a different one. I'm not able to reproduce the same error.

![CleanShot 2021-06-18 at 16 16 21](https://user-images.githubusercontent.com/240569/122559647-ab031880-d050-11eb-9a78-d4f7b5811169.png)

**Comment by optimizewp** [comment]
Thanks @Mamaduka, just as a confirm, previous screenshot is with `WP 5.8-beta1`, `TT1 Blocks` theme and with Edge Version 91.0.864.48 (Official build) (64-bit). 

Gutenberg downloaded from here 

* https://github.com/WordPress/gutenberg/releases/tag/v10.9.0-rc.1
* https://github.com/WordPress/gutenberg/archive/refs/tags/v10.9.0-rc.1.zip

The file is `/wp-content/plugins/gutenberg/build/edit-site/index.min.js`, which seems to be the same as yours. But it has 26 lines. 

I get the same error with Chrome Version 91.0.4472.114 (Official Build) (64-bit) and Firefox DE 90.0b9 (64-bit).

Hopes this helps reproducing it.

**Comment by optimizewp** [comment]
Reported issue is still present, same error, also with WordPress 5.8 beta 3 just released.

**Comment by torounit** [comment]
@optimizewp get any PHP warnings when I access wp-admin?

**Comment by torounit** [comment]
I was able to reproduce this problem.

If I install https://github.com/WordPress/gutenberg/releases/download/v10.9.0-rc.1/gutenberg.zip in an environment where error_reporting is enabled, I get the following error .

````
file_get_contents(C:\Users\torounit\Local Sites\gbtest\app\public\wp-content\plugins\gutenberg\lib/. /build/widgets/blocks/legacy-widget/block.json): failed to open stream: No such file or directory in public\wp-content\plugins\gutenberg\lib\blocks.php on line 127
````

```
Cannnot modify heder inofrmation....
```

Isn't this error message preventing the JavaScript from loading properly?

Tested on: Local by Flyweel
OS: Windows 10
Browser: edge / chorme

**Comment by torounit** [comment]
@optimizewp 

Does this problem still occur with error_reporting disabled?

I think you can try by setting WP_DEBUG to false and putting `ini_set( 'display_errors', 0 );` in wp-config.php.

**Comment by optimizewp** [comment]
@torounit 

yeah, I had `WP_DEBUG` set to true, with `define( 'WP_DEBUG_DISPLAY', true );` too. And the issue you just described, already reported here https://github.com/WordPress/gutenberg/issues/32785 is still actual on Gutenberg `10.9.0 RC1`, so that was breaking `Site Editor`.

Setting  `define( 'WP_DEBUG_DISPLAY', false );` makes `Site Editor` working back for me too.

Good catch! Thanks for it.

For sure when #32785 patch will be live in next Gutenberg release, this notification will be gone, and this issue will be solved.

But this does not prevent other errors to appear in the future, and I'll be not the only one with error_reporting enabled.

I'll let you decide if this requires some attention and a fix, or simply acknowledge it, and close this ticket.

Thanks again :bow:

**Comment by optimizewp** [comment]
`Gutenberg 10.9.0` has just been released, tested in the same install, with WP 5.8-beta3.

Previously reported warning has been fixed, now if we enable `error_reporting` we have this new one:

```php
Warning: Block core/query-loop has been renamed to Post Template. core/query-loop will be supported until WordPress version 5.9. in \wp-includes\blocks\post-template.php on line 107
```

displaying on `Site Editor` loading, while you can also see the wp-admin toolbar and menu flashing for a second, but this does not prevent the editor to fully load, so everything works fine and I cannot see any blocker with this issue anymore.

Feel free to acknowledge this, and close this ticket, if you are happy.

**Comment by torounit** [comment]
@optimizewp 

Yes, I agree with you. I should only hide errors when I am sure that hiding them is causing the problem.

Thank you for taking the time to this issue.
